src/arviz_plots/plots/bfplot.py

```diff
@@ -0,0 +1,120 @@
+import logging
+from arviz.plots.plot_utils import get_plotting_function
```

Comment:
You can not use ArviZ. We final goal of the refactoring is that the ArviZ package you are importing will not exist anymore.

The rest of the implementation is not how plots are implemented in Arviz-plots. Please familiarize yourself with the library before adding more changes.  See how other plots are implemented, like plot_energy or plot_psense_dist